docs/2.guide/3.going-further/3.modules.md

```diff
@@ -411,6 +411,37 @@ export default defineNuxtModule({
 })
 ```
 
+#### Using other modules in your module
+
+If your module depends on other modules, you can add them by using `installModule`. For example, suppose you want to use `tailwindcss` framework in your module, you can add it as below:
+
+```ts
+import { defineNuxtModule, createResolver, installModule } from '@nuxt/kit'
+
+export default defineNuxtModule<ModuleOptions>({  
+  setup (options, nuxt) {
+    const { resolve } = createResolver(import.meta.url)
+
+    // We can inject our css file which includes tailwind's directives
+    nuxt.options.css.push(resolve('./runtime/assets/styles.css'))
+
+    installModule('@nuxtjs/tailwindcss', {
```

Comment:
It is really important to always `await` on install module otherwise Nuxt lifecycle get's own of the order

```suggestion
    await installModule('@nuxtjs/tailwindcss', {
```